packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -0,0 +1,219 @@
+import {LightningElement, api} from 'lwc';
+import {getAllFacetsFromStore} from 'c/quanticHeadlessLoader';
+import {
+  initializeWithHeadless,
+  registerComponentForInit,
+} from 'c/quanticHeadlessLoader';
+import filters from '@salesforce/label/c.quantic_Filters';
+import clearAllFilters from '@salesforce/label/c.quantic_ClearAllFilters';
+
+/** @typedef {import("coveo").SearchStatus} SearchStatus */
+/** @typedef {import("coveo").SearchEngine} SearchEngine */
+/** @typedef {import("coveo").BreadcrumbManager} BreadcrumbManager */
+
+const COMMON_FACET_PROPERTIES = ['facetId', 'field', 'label'];
+const DEFAULT_FACET_PROPERTIES = [
+  'numberOfValues',
+  'sortCriteria',
+  'noSearch',
+  'displayValuesAs',
+  'noFilterFacetCount',
+  'injectionDepth',
+];
+const NUMERIC_FACET_PROPERTIES = [
+  'numberOfValues',
+  'sortCriteria',
+  'rangeAlgorithm',
+  'withInput',
+  'formattingFunction',
+];
+const CATEGORY_FACET_PROPERTIES = [
+  'basePath',
+  'noFilterByBasePath',
+  'noFilterFacetCount',
+  'delimitingCharacter',
+  'numberOfValues',
+  'sortCriteria',
+  'withSearch',
+];
+const TIMEFRAME_FACET_PROPERTIES = [
+  'withDatePicker',
+  'noFilterFacetCount',
+  'injectionDepth',
+];
```

Comment:
Is there a way other facets could export those? Would we want to handle it like that instead?

Reply:
We can the properties as static properties in each facets to get something like this:
<img width="631" alt="static" src="https://user-images.githubusercontent.com/86681870/167421029-b6ee9db9-9fc9-49f0-9926-7bbc6717acc5.png">
and then then we can use them in the QuanticRefineModalContent component as follows:
`const NUMERIC_FACET_PROPERTIES = QuanticNumericFacet.attributes;`

What do you think?